File: src/main/java/org/example/chessplatformbe/controller/dto/request/ChatMessageDTO.java
package org.example.chessplatformbe.controller.dto.request;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class ChatMessageDTO {
    private String content;
    private String sender;
    private Integer streamId;
}

File: src/controller/dto/request/chat_message_dto.rs
pub struct ChatMessageDTO {
    content: String,
    sender: String,
    stream_id: u32,
}

impl ChatMessageDTO {
    pub fn new(content: String, sender: String, stream_id: u32) -> Self {
        Self {
            content,
            sender,
            stream_id,
        }
    }
}
